tests/component-tests/textfieldControl/js/TextfieldStepperTests.js

```diff
@@ -0,0 +1,209 @@
+            jqUnit.assertEquals("The state of the button should match the model", that.model.disabled, that.container.is(":disabled"));
+        });
+
+        jqUnit.test("Change State", function () {
+            var that = fluid.tests.textfieldStepper.button(".flc-textfieldStepper-button");
+            var newState = !that.model.disabled;
+
+            that.applier.change("disabled", newState);
+            jqUnit.assertEquals("The state of the button should match the model", newState, that.container.is(":disabled"));
+        });
+
+        jqUnit.test("Trigger Click", function () {
+            jqUnit.expect(1);
+            var that = fluid.tests.textfieldStepper.button(".flc-textfieldStepper-button", {
+                listeners: {
+                    "onClick.test": {
+                        listener: "jqUnit.assert",
+                        args: ["The onClick event should have fired."]
+                    }
+                }
+            });
+
+            that.container.trigger("click");
+        });
+
+        /***************************
+         * Textfield Stepper Tests *
+         ***************************/
+
+        jqUnit.module("Textfield Stepper Tests");
+
+        fluid.defaults("fluid.tests.textfieldStepper", {
+            gradeNames: ["fluid.textfieldStepper"],
+            strings: {
+                "aria-label": "Aria self-labeling"
+            },
+            model: {
+                value: 0
+            },
+            ariaOptions: {
+                "aria-labelledby": "label-nativeHTML"
+            }
+        });
+
+        jqUnit.test("Test Init", function () {
+            jqUnit.expect(11);
+            var options = {
+                model: {
+                    value: 8,
+                    range: {
+                        max: 10,
+                        min: 1
+                    }
+                }
+            };
+            var that = fluid.tests.textfieldStepper(".flc-textfieldStepper", options);
+
+            fluid.tests.textfieldControl.assertRangeControlledTextfieldInit(that.textfield, options);
+            jqUnit.assertEquals("The role is set", "spinbutton", that.textfield.container.attr("role"));
+            jqUnit.assertEquals("The aria-valuemin is set", that.model.range.min.toString(), that.textfield.container.attr("aria-valuemin"));
+            jqUnit.assertEquals("The aria-valuemax is set", that.model.range.max.toString(), that.textfield.container.attr("aria-valuemax"));
+            jqUnit.assertEquals("The aria-valuenow is set", that.model.value.toString(), that.textfield.container.attr("aria-valuenow"));
+
+            jqUnit.assertFalse("The increase button is enabled", that.locate("increaseButton").is(":disabled"));
+            jqUnit.assertFalse("The decrease button is enabled", that.locate("decreaseButton").is(":disabled"));
+        });
+
+        jqUnit.test("Test Maximum", function () {
```

Comment:
The test sets for "Test Maximum" and "Test Minimum" are quite similar. Please create a common utility to reuse.